docs/Tutorial.md

```diff
@@ -264,15 +264,15 @@ To achieve this, here a little example:
 pdf = PDF()
```

Comment:
I'm not sur that this code block nor the `fpdf2-logo` above are really needed.

Same in the German tutorial 🙂